test/unit/Console/VersionProviderTest.php

```diff
@@ -32,7 +32,7 @@ public function testComposerInstalledVersion()
         $this->assertInternalType('string', $actual, 'Version of phpunit package was found installed');
 
         // dev-master is included here as the phpunit package is checked and there is a dev-master used on travis
-        $this->assertRegExp("~^dev-master|\d.\d.\d+$~", $actual, 'Actual version number');
+        $this->assertRegExp("~^dev-master|\d.\d.(.)+$~", $actual, 'Actual version number');
```

Comment:
phpunit is not massively consistent reporting its version numbers. This matches 6.5.x-dev (current)